src/common.jl

```diff
@@ -13,7 +13,7 @@ function update!(context::T, data::U) where {T<:SHA_CTX,
     usedspace = context.bytecount % blocklen(T)
     while len - data_idx + usedspace >= blocklen(T)
         # Fill up as much of the buffer as we can with the data given us
-        copy!(context.buffer, usedspace + 1, data, data_idx + 1, blocklen(T) - usedspace)
+        copyto!(context.buffer, usedspace + 1, data, data_idx + 1, blocklen(T) - usedspace)
```

Comment:
if this isn't in a compat release yet, you really shouldn't tag a release here using it. if it is in a release, then the bound in require should reflect that.

Reply:
Ah, I thought that `0.38` contained `copyto!()` but it's actually `0.45`.  Thanks.

Reply:
and what about `GC.` - doesn't look like that's in Compat at all

Reply:
Yep; I'm happy to have the `perf` script be 0.7-only for now.